plexus-compilers/plexus-compiler-javac/src/main/java/org/codehaus/plexus/compiler/javac/InProcessCompiler.java

```diff
@@ -3,8 +3,9 @@
 import org.codehaus.plexus.compiler.CompilerConfiguration;
 import org.codehaus.plexus.compiler.CompilerException;
 import org.codehaus.plexus.compiler.CompilerResult;
+import org.codehaus.plexus.logging.LogEnabled;
 
-public interface InProcessCompiler {
+public interface InProcessCompiler extends LogEnabled {
```

Comment:
I had to change that to be able to get logger.
The only usage of this in project is a manual construction of the only implementation available here and I bump version number.